tools/codegen/api/translate.py

```diff
@@ -44,14 +44,33 @@ class UnsatError(RuntimeError):
 #   }
 #
 # and you need to generate "exprs".
-#
-# TODO: Don't need full Binding for goals, CType will do
-# TODO: Don't need full Binding for bindings, list of Expr will do
```

Comment:
I think it's a short comment header documenting the params - even just the signal in these TODOs would be helpful to someone who's unfamiliar with this code